docs/shortfin/llm/user/shortfin_with_sglang_frontend_language.md

```diff
@@ -24,21 +24,15 @@ For this tutorial, you will need to meet the following prerequisites:
     - You can check out [pyenv](https://github.com/pyenv/pyenv)
     as a good tool to be able to manage multiple versions of python
     on the same system.
-- A running `shortfin` LLM server as described [below](#installstart-shortfin-llm-server)
+
+### Shortfin LLM Server
+
+- A running `shortfin` LLM server. Directions on launching the llm server on one system can be found [here](./llama_end_to_end.md) and for launching
+on a kubernetes cluster, please look [here](./e2e_llama8b_k8s.md)
```

Comment:
docs style tips: https://developers.google.com/style/link-text

> To write link text, use short, descriptive phrases that provide context for the material that you're linking to.

> Write unique, descriptive link text that makes sense without the surrounding text. Don't use phrases such as _this document_, _this article_, or _click here_.

(and "here" is only four characters, making it difficult to click on when on mobile)

Here is one way to rephase, by inlining the document titles:

```suggestion
- A running `shortfin` LLM server. Directions on launching the llm server on one system can be found in [Llama end to end serving instructions](./llama_end_to_end.md) and for launching
on a kubernetes cluster, see [LLama 8b GPU instructions on Kubernetes](./e2e_llama8b_k8s.md)
```